stac_check/lint.py

```diff
@@ -33,6 +36,36 @@ def __post_init__(self):
         self.file_name = os.path.basename(self.item).split('.')[0]
         self.best_practices_msg = self.create_best_practices_msg()
 
+    def parse_config(self, config_file):
+        default_config = {
+            'linting': {
+                'searchable_identifiers': True, 
+                'percent_encoded': True,
+                'item_id_file_name': True,
+                'catalog_id_file_name': True,
+                'check_summaries': True,
+                'null_datetime': True,
+                'check_unlocated': True,
+                'check_geometry': True,
+                'bloated_links': True,
+                'bloated_metadata': True,
+                'check_thumbnail': True,
+                'links_title': True,
+                'links_self': True,
+            }
+        }
+        if config_file == False:
+            config_file = "stac-check.config.yml"
+        try:
+            with open(config_file) as f:
+                config = yaml.load(f, Loader=yaml.FullLoader)
+        except Exception:
+            config = {}
+
+        default_config.update(config)
+
+        return default_config
```

Comment:
This should not be an instance method (i.e. it should not take `self`) as it doesn't use any methods or attributes of `self`.

Reply:
Ah ok

Reply:
The function is called in post_init ```self.config = self.parse_config(self.config_file)```